packages/dd-trace/src/plugins/tracing.js

```diff
@@ -69,6 +70,11 @@ class TracingPlugin extends Plugin {
       childOf = store.span
     }
 
+    if (this.constructor.name && (!meta || !meta.component)) {
```

Comment:
Why not just always set it to the value of the component property of the plugin? The idea is to avoid having to set the tag.

Reply:
Didn't see that the Tracingplugin class has a component field getting set within this file. I changed this code to set the component tag on a span if the component field is not Null.